src/javascript/app/japan/knowledge_test/knowledge_test.ui.js

```diff
@@ -4,25 +4,29 @@ const urlForStatic = require('../../../_common/url').urlForStatic;
 
 const KnowledgeTestUI = (() => {
     const createTrueFalseBox = (question, show_answer) => {
-        const qid      = question.id;
-        const true_id  = `${qid}true`;
-        const false_id = `${qid}false`;
+        const qid               = question.id;
+        const true_id           = `${qid}true`;
+        const false_id          = `${qid}false`;
+        const center_text_class = 'center-text';
 
         const $true_button = $('<input />', {
             type : 'radio',
             name : qid,
             id   : true_id,
             value: '1',
         });
-        const $true_td     = $('<td></td>').append($true_button);
+
+        const $true_el = $true_button.add(`<label for=${true_id}>&#8203</label>`);
+        const $true_td = $(`<td class=${center_text_class}></td>`).append($true_el);
```

Comment:
can we create these jquery elements the same way as on line 12? so something like `$('<td />', { class: center_text_class })`